cmd/api.go

```diff
@@ -206,7 +206,7 @@ func (o *apiOptions) validate(c *config.Config) error {
 
 func (o *apiOptions) scaffolder(c *config.Config) (scaffold.Scaffolder, error) {
 	// Load the boilerplate
-	bp, err := ioutil.ReadFile(filepath.Join("hack", "boilerplate.go.txt")) // nolint:gosec
+	bp, err := ioutil.ReadFile(filepath.Join("hack", "boilerplate.go.txt")) //nolint:gosec
```

Comment:
What is the complaint that is fixed with this space?

Reply:
See https://github.com/kubernetes-sigs/kubebuilder/pull/1368#discussion_r379111906 where @hypnoglow provided the link to the documentation where it states that spaces should not be used.